pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -296,21 +290,26 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
+func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
```

Comment:
kubelet has a util function that does similar stuff. But probably we should not import it into this package?

Reply:
@freehan I did see PodFullName() but that doesn't do the containerID too.  Which one were you thinking of?

Reply:
That was the one I am looking at. Never mind then.